bindings_ffi/Cargo.toml

```diff
@@ -22,6 +22,8 @@ xmtp_api_grpc = { path = "../xmtp_api_grpc" }
 xmtp_v2 = { path = "../xmtp_v2" }
 futures = "0.3.28"
 tokio = { version = "1.28.1", features = ["macros"] }
+xmtp_proto = { path = "../xmtp_proto", features = ["proto_full"] }
```

Comment:
You should be able to get away with just setting `features = ["xmtp-message_api-v1"]`. That'll give it a smaller footprint.